### ISSUE TITLE
Required Properties Missing at Top Level?

### ISSUE THREAD

**Issue by joewlambeth** [bids-standard/legacy-validator#912]
I have an EEG based BIDS project, where roughly half of the subjects have one value for "SamplingFrequency", and the other half have another. Therefore, "SamplingFrequency" is specified at each subject-level sidecar. However, having that approach yields this error:

```
./task-XXX_eeg.json
    Evidence: should have required property 'SamplingFrequency'
```

I'm not convinced that this is a specification issue, since [this clause](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#good-practice-recommendations) backs up my formatting.

**Comment by effigies** [comment]
I agree with this. Generally, the rule should be that an eeg data file should have a `SamplingFrequency` metadata associated, not that a eeg sidecar file should have it included.

I think we do this for `RepetitionTime` and `_bold.nii[.gz]` files, so there should be precedent we can follow.

**Comment by VisLab** [comment]
Has any progress been made on this?

The issue is we have a study where different EEG files have different sampling frequencies.  The x_eeg.json file at the top level has all of the common parameters specified, but not samplingFrequency.  Instead, this is specified in the x_eeg.json one level down below in the sub_y subdirectory (along with EEGChannelCount to override the one at the top level).  It doesn't seem to be recognizing that it should override values as you traverse into the subdirectory. I think this is a parsing error in the validator.

**Comment by VisLab** [comment]
I'm trying to get a response on this.  The bids-validator has a JSON_SCHEMA_VALIDATION_ERROR because the samplingFrequency is missing from the top-level _eeg.json.   My reading of the bids-specification is that sidecar values are taken by the version closest to the bottom level.  Do you need to put "dummy"  samplingFrequency all the way down the tree?  If so, it should be clarified in the bids-specification.   Thanks.....

**Comment by sappelhoff** [comment]
@rwblair have you had time to look at this? I also think that this is an error with the validator and not with the specification.

@VisLab I do not think that dummy values have to be put all the way down the tree.

**Comment by rwblair** [comment]
Confirmed it as a validator issue. Looks like the reason why RepetitionTime gets away with it is because it is not listed as a required key in the bold json schema.

I'm working on finding where and how the schema is being applied to just the top level json and not the full merged list of sidecars.
